test/src/main/java/org/apache/accumulo/test/functional/AccumuloClientIT.java

```diff
@@ -220,6 +238,120 @@ public void testClose() throws Exception {
     expectClosed(() -> tops.create("expectFail"));
     expectClosed(() -> tops.cancelCompaction(tableName));
     expectClosed(() -> tops.listSplits(tableName));
+  }
 
+  @Test
+  public void testAmpleReadTablets() throws Exception {
```

Comment:
Why put this test in AccumuloClientIT?

Reply:
This is here to facilitate calling the method from a AccumuloClient.